### DISCUSSION TITLE
Reset library / delete tracks from after provider removal

### DISCUSSION THREAD

**Discussion by ministryofsillywalks** [music-assistant/support#2068] [Q&A] [answered]
I had two different spotify accounts setup in MA however I deleted one of the accounts but the associated albums and playlists still show up in my library.
Is there any way to easily rescan the library or reset it so that it automatically notices that the tracks shouldn't be present anymore?

**Comment by OzGav** [comment, marked as answer]
If you restarted MA after removing the provider but before the cleanup was completed then it is now not possible. You will likely have to completely remove MA and start again although you could try adding the Spotify account again waiting for it to fully sync and then remove it again.

if you haven’t restarted MA then it is likely a bug

**Reply by ministryofsillywalks** [reply]
Thanks for the info.
As I had multiple restarts since removing the account, I ended up uninstalling and starting from scratch.
